.gitignore

```diff
@@ -8,3 +8,4 @@
 dependency-reduced-pom.xml
 **/nbproject/
 **/access.log
+carapace-server/dbdata/*
```

Comment:
Is this needed?

Reply:
With this commit tests about persistence on herddb now uses a temp dir, so it's useless indeed. I'll remove it.